README.md

```diff
@@ -31,9 +31,9 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.datasource.driver-class-name=org.postgresql.Driver
 
 #Database properties
-spring.datasource.url=jdbc:postgresql://localhost:5432/yourDatabaseName
+spring.datasource.url=jdbc:postgresql://localhost:5490/board-game-training-db
```

Comment:
As @Montagon said below, this information is not necessary right now, we can remove it from the README 

We only can remember to the user if he need to change something related to the database, he should change it in the `application.properties` file